lib/ansible/cli/galaxy.py

```diff
@@ -957,9 +975,13 @@ def execute_remove(self):
 
     def execute_list(self):
         """
-        lists the roles installed on the local system or matches a single role passed as an argument.
+        lists the roles or collections installed on the local system or matches a single role or collection passed as an argument.
         """
 
+        def _display_collection(collection):
+            fqcn = '%s.%s' % (collection.namespace, collection.name)
+            display.display('- {fqcn} ({version})'.format(fqcn=fqcn, version=collection.latest_version))
```

Comment:
I would like them to be consistent. If anything, though, I would like to change the output of `role list` because it's not great.

I did the work of making the formats match, but I've moved the role output changes to a separate branch for now.